test/sample-test.ts

```diff
@@ -2,12 +2,47 @@ import { expect } from "chai";
 import { starknet } from "hardhat";
 import { StarknetContract, StarknetContractFactory } from "hardhat/types/runtime";
 
+const starkwareCrypto = require('../lib/crypto/starkware/crypto/signature/signature.js');
+
 describe("Starknet", function () {
   this.timeout(300_000); // 5 min
   let preservedAddress: string;
 
   let contractFactory: StarknetContractFactory;
 
+  // Amount variables
+  const balance_amount = 50;
+  const bet_amount = 10;
+  const balance_ammount_message_hash = starkwareCrypto.pedersen([balance_amount,0]);
+  const bet_amount_message_hash = starkwareCrypto.pedersen([bet_amount]);
+
+  //Key definition and generation
+  const bettor_private_key = 12345;
+  const counter_bettor_private_key = 54321;
+  const judge_private_key = 121212;
+
+  const bettor_key_pair = starkwareCrypto.ec.keyFromPrivate(bettor_private_key, 'hex');
+  const bettor_public_key = starkwareCrypto.ec.keyFromPublic(bettor_key_pair.getPublic(true, 'hex'), 'hex');
+  
+  
+  const counter_bettor_key_pair = starkwareCrypto.ec.keyFromPrivate(counter_bettor_private_key, 'hex');
+  const counter_bettor_public_key = starkwareCrypto.ec.keyFromPublic(counter_bettor_key_pair.getPublic(true,'hex'), 'hex');
+  
+  
+  const judge_key_pair = starkwareCrypto.ec.keyFromPrivate(judge_private_key, 'hex');
+  const judge_public_key =   starkwareCrypto.ec.keyFromPublic(judge_key_pair.getPublic(true,'hex'), 'hex');
+
+  //Message hash definition
+  const balance_signature_bettor = starkwareCrypto.sign(bettor_key_pair, balance_ammount_message_hash);
+  const balance_signature_counter_bettor = starkwareCrypto.sign(counter_bettor_key_pair, balance_ammount_message_hash);
+
+  const bet_amount_signature_bettor = starkwareCrypto.sign(bettor_key_pair,bet_amount_message_hash)
+
+  const bettor_pub_key = "0x"+bettor_public_key.pub.getX().toString(16);
```

Comment:
I notice some variable names not indicative of their purpose. E.g. if after some time you come back to this code, you might forget the distinction between variables like `bettor_public_key` and `bettor_pub_key`.